winter-website.css

```diff
@@ -0,0 +1,243 @@
+html, body {
+	padding: 0px;
+	margin: 0px;
+	color: #e8f3fc;
+	background-color: #000000;
+	font-family: sans-serif;
+	font-size: 10px;
+	height: 100%;
+}
+
+*, *::before, *::after {
+   box-sizing: border-box;
+}
+
+.navBar {
+	display: flex;
+	justify-content: space-between;
+	width: auto;
+	padding: 20px;
+	margin: 20px;
+	background-color: #5d87cf;
+	opacity: 0.5;
+	border-radius: 0.6rem;
+	border-style: solid;
+	border-width: 0.15rem;
+	border-color: #d4bdff;
+}
+
+a.navLink {
+	display: flex;
+	color: #e8f3fc;
+	text-decoration: none;
+	font-size: 2rem;
+	background-color: #856ab8;
+	padding: 1rem;
+	border-radius: 0.4rem;
+	border-style: solid;
+	border-width: 0.15rem;
+	border-color: #d4bdff;
+}
+
+.navLink:hover {
+	color: #000000;
+	background-color: #d4bdff;
+}
+
+h1 {
+	font-size: 10rem;
+	padding: 10px;
+	margin: 0;
+}
+
+.sectionAll {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	flex-wrap: nowrap;
+	width: auto;
+	padding: 2%;
+	margin: 20px;
+}
+
+.aboutMeSection {
+	min-width: 200px;
+	width: 40%;
+	padding: 20px;
+	margin: 20px;
+}
+
+h2 {
+	padding: 0;
+	margin: 0;
+	font-size: 2rem;
+}
+
+.aboutMeImages {
+	display: flex;
+	flex-wrap: wrap;
+	min-width: 500px;
+	max-width: 600px;
+	padding: 20px;
+	margin: 20px;
+	justify-content: center;
+}
+
+.aboutMeImages img {
+	width: 40%;
+	height: auto;
+	padding: 10px;
+}
+
+.footerBar {
+	width: auto;
+	color: #e8f3fc;
+	opacity: 0.75;
+}
+
+@media screen and (max-width: 1100px) {
+	h1 {
+		font-size: 8rem;
+		padding: 0;
+		margin: 0;
+	}
+	
+	h2 {
+		padding: 0;
+		margin: 0;
+		font-size: 1.5rem;
+	}
+}
+
+@media screen and (max-width: 990px) {
+	h1 {
+		font-size: 6rem;
+		padding: 0;
+		margin: 0;
+	}
+	
+	h2 {
+		padding: 0;
+		margin: 0;
+		font-size: 1.5rem;
+	}
+}
+
+@media screen and (max-width: 760px) {
+	
+	.navBar {
+		display: flex;
+		flex-direction: row;
+		opacity: 0.75;
+	}
+	
+	h1 {
+		font-size: 6rem;
+		padding: 0;
+		margin: 0;
+	}
+	
+	h2 {
+		padding: 0;
+		margin: 0;
+		font-size: 1.5rem;
+	}
+	
+	.sectionAll {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		flex-wrap: wrap;
+		width: auto;
+		padding: 20px;
+		margin: 20px;
```

Comment:
2rem

Reply:
well i still reckon rem

Reply:
as discussed i left the rem suggestions as they are because the values work better as percentages and other measurements : ]